modules/local/multiqc/main.nf

```diff
@@ -1,10 +1,10 @@
 process MULTIQC {
     label 'process_medium'
 
-    conda "bioconda::multiqc=1.15"
+    conda "bioconda::multiqc=1.17"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/multiqc:1.15--pyhdfd78af_0' :
-        'biocontainers/multiqc:1.15--pyhdfd78af_0' }"
+        'https://depot.galaxyproject.org/singularity/multiqc:1.17--pyhdfd78af_0' :
+        'biocontainers/multiqc:1.17--pyhdfd78af_0' }"
```

Comment:
Build 1 is already available, but I do not know if there is any difference.

BTW: [In terms of security, the container doesn't look too well maintained](https://quay.io/repository/biocontainers/multiqc/manifest/sha256:fb7d6625fb5adaed43ced8bd051a875038714180bcfcd7c8e467204f72882de9?tab=vulnerabilities).
```suggestion
        'https://depot.galaxyproject.org/singularity/multiqc:1.17--pyhdfd78af_1' :
        'biocontainers/multiqc:1.17--pyhdfd78af_1' }"
```